content/en/continuous_integration/setup_tests/ruby.md

```diff
@@ -61,16 +61,16 @@ require 'datadog/ci'
 
 Datadog.configure do |c|
   # Only activates test instrumentation on CI
-  c.tracer.enabled = (ENV["DD_ENV"] == "ci")
+  c.tracing.enabled = (ENV["DD_ENV"] == "ci")
 
   # Configures the tracer to ensure results delivery
-  c.ci_mode.enabled = true
+  c.ci.enabled = true
```

Comment:
I'm a bit lost on details right now. If tracing is disabled, it does not really matter if ci is enabled, does it? We won't get any tests. I think the idea behind `c.tracing.enabled = (ENV["DD_ENV"] == "ci")` was to make sure we don't activate ci in prod environments, right?

Reply:
I think you're right.

I guess the confusion for me comes from there being three supported configurations--- ✅ tracing enabled in prod / ✅ ci enabled in ci, ❌ tracing disabled in prod / ✅ ci enabled in ci and ✅ tracing enabled in prod / ❌ ci disabled in ci -- and we end up using different variants in different documents but without ever stating which variant we're doing?

So here we're doing ❌ tracing disabled in prod / ✅ ci enabled in ci, but for instance in the web UI (which we were just discussing in https://github.com/DataDog/web-ui/pull/56664) we're actually doing ✅ tracing enabled in prod / ✅ ci enabled in ci.

This gets complicated further by the fact that "having tracing enabled" is a semi-NO-OP unless the customer actually enables instrumentations or uses the APIs manually.

Reply:
I think the setup in this documentation file has to be rethought-out.

`c.tracing.enabled = (ENV["DD_ENV"] == "ci")` is incompatible with a production environment, as it will remove the ability for users to the disable the tracer with the environment variable `DD_TRACE_ENABLED`.

The snippets in this file work well in a CI-only `ddtrace` installation, but shouldn't be executed in a production environment.

I think it's probably outside of the scope of this PR to address this: I can't think of an uncontroversial solution, but open to suggestion of course. I've left a comment documentation the issue in code.